pypeit/scripts/setup_coadd2d.py

```diff
@@ -170,7 +177,8 @@ def main(args):
         for obj, files in object_spec2d_files.items():
             tbl = Table()
             tbl['filename'] = [f.name for f in files]
-            ofile = args.pypeit_file.replace('.pypeit', f'_{obj}.coadd2d')
-            inputfiles.Coadd2DFile(config=cfg, file_paths=[str(sci_dir)],
+            ofile = args.pypeit_file.replace('.pypeit', f'_{obj}.coadd2d') if args.pypeit_file is not None \
+                else f'{spec_name}_{obj}.coadd2d'
```

Comment:
Nice!  Can we add a test in the dev-suite?  There is a test in `vet_tests/test_scripts.py` called `test_setup_coadd2d`; can you add a call that doesn't include the pypeit file?

Reply:
Done. Here it is:https://github.com/pypeit/PypeIt-development-suite/pull/271